Jenkinsfile

```diff
@@ -29,6 +30,73 @@ pipeline {
                 sh 'build/run make -j\$(nproc) test-integration'
             }
         }
+
+        stage('Integration Tests SetUp') {
```

Comment:
can you combine this with parallel runs stage. we just need one "integration tests" stage